monitor/src/main/webapp/resources/monitor/reload.js

```diff
@@ -0,0 +1,23 @@
+window.setInterval(function() {
+	$.ajax({
+		type : "GET",
+		dataType : "json",
+		url : "admin/apps/reload",
+		success : function(data) {
+			data.forEach(update)
+		},
+		error : function(xhr, status, error) {
+			console.error(xhr.responseText)
+			console.error(status)
+			console.error(error.message)
+			alert('This page could not be updated')
```

Comment:
let's make this a bootstrap alert that says something like "Connection to server lost. Trying to connect..." or something like that. And then hide the alert if it can connect again.